### ISSUE TITLE
`unresolve comment` does not move focus to it

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#210090]
1. resolve a comment
2. use the context menu action, `unresolve comment` in the comment panel 
3. expected: focus of the comment, actual: focus doesn't move to the comment

**Comment by meganrogge** [comment]
https://github.com/microsoft/vscode/assets/29464607/9ab6ac5a-a4ab-4ac0-8799-50fecf63c67e

**Comment by alexr00** [comment]
Not intentional and inconsistent with the way other trees work.

**Comment by alexr00** [comment]
I can't actually repro that the comment thread in the tree view doesn't get focus:
1. Put focus in the editor
2. Right click on a comment thread in the comments view
3. "Unresolve" from the context menu
4. The comment thread in the tree view gets focus. 

Is this what you're seeing @meganrogge?

**Comment by meganrogge** [comment]
I am not focusing the editor first. 

1. right click on a comment thread in the comments view
2. resolve conversation
3. scroll away 
4. right click on the comment thread in the comments view
5. unresolve
6. 🐛 it doesn't scroll that comment into view
https://github.com/microsoft/vscode/assets/29464607/be011c2e-b8fb-41ef-857c-4deb73aa644b

**Comment by alexr00** [comment]
I think this behavior is correct. When a tree is taken, the focus should remain in the tree.

**Comment by meganrogge** [comment]
I agree the focus should remain in the tree. I had just expected the comment widget that was unresolved to be revealed.

**Comment by alexr00** [comment]
I get it now: reveal the comment in the editor, not focus it. 

This is interesting. Is it just the "unresolve" action that you would expect to reveal the comment in the editor? I'm thinking there are some actions it makes sense to reveal for, and some that it doesn't.

**Comment by meganrogge** [comment]
I am not sure which other actions you are thinking of. But I think since a user is unresolving the comment, it implies they might take an action on it, so revealing makes sense IMO.
